yolov8_recognition:
  ros__parameters:

    threshold: 0.6

    start_on_init: True
      
    max_sizes:
        - [0.05, 0.05, 0.05]

    model_file: yolov8n.pt

    services:
    
      object_recognition:
        start: /fbot_vision/fr/object_start
        stop: /fbot_vision/fr/object_stop

    subscribers:
      
      slop: 0.2
      qos_profile: 10

      image_rgb: /cam1/color/image_raw
      camera_info: /cam1/color/camera_info
      image_depth: /cam1/aligned_depth_to_color/image_raw
      
    publishers:

      object_recognition:
        topic: /fbot_vision/fr/object_recognition
        qos_profile: 10

      debug:
        topic: /fbot_vision/fr/object_debug
        qos_profile: 10